Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/transform/AuditTransforms.java

```diff
@@ -480,10 +480,12 @@ private static AuditSourceIdentificationType createAuditSourceIdentification(Str
 
         // Set the Audit Source Id (community id)
         if (communityId != null) {
-            if (communityId.startsWith("urn:oid:")) {
-                auditSrcId.setAuditSourceID(communityId.substring(8));
-            } else {
+            if (communityId.startsWith(AuditTransformsConstants.HOME_COMMUNITY_ID_PREFIX)) {
```

Comment:
We can use HomeCommunityMap.getHomeCommunityIdWithPrefix(String communityId) function for getting the HCID with prefix.